runtime/src/snapshot_archive_info.rs

```diff
@@ -65,6 +65,10 @@ impl FullSnapshotArchiveInfo {
     pub(crate) fn new(snapshot_archive_info: SnapshotArchiveInfo) -> Self {
         Self(snapshot_archive_info)
     }
+
+    pub fn is_remote(&self) -> bool {
+        self.path().parent().unwrap().ends_with("remote")
+    }
```

Comment:
1. I'd rather see this function return `false` in case `.parent()` fails, instead of crashing.

2. I think this fn should be part of the SnapshotArchiveInfoGetter trait, instead of just on the `FullSnapshotArchiveInfo` struct.

Reply:
yeah.